gimme

```diff
@@ -209,16 +212,16 @@ _compile() {
 		export CC_FOR_TARGET="${GIMME_CC_FOR_TARGET}"
 
 		local make_log="${1}/make.${GOOS}.${GOARCH}.log"
-		if [ "${GIMME_DEBUG}" > "1" ]; then
-			./make.bash 2>&1 | tee $make_log 1>&2 || return 1
+		if [ "${GIMME_DEBUG}" ] >"1"; then
```

Comment:
This is exposing a bug in the existing code.

`[` does not change parser state, so `>` within `[ ... ]` is a shell-redirection, so you redirected to create a file named `1`

You want `[[ ... > ... ]]` using the lexical-parse-changing doubled-square brackets to keep the string comparison, but I suspect you want `-gt` for numerical comparison (which should work with either singled or doubled square brackets), or just `if (( GIMME_DEBUG > 1 )); then` for arithmetic comparison.